core/db/entities.py

```diff
@@ -35,3 +40,72 @@ class Contact(db.Model):
     telephone = sqla.Column(sqla.String, nullable=True)
 
     organisation = sqla.orm.relationship("Organisation", back_populates="contacts")
+
+    name_packages = sqla.orm.relationship(
+        "Package", back_populates="name_contact", foreign_keys="Package.name_contact_id"
+    )
+    project_sro_packages = sqla.orm.relationship(
+        "Package",
+        back_populates="project_sro_contact",
+        foreign_keys="Package.project_sro_contact_id",
+    )
+    cfo_packages = sqla.orm.relationship("Package", back_populates="cfo_contact", foreign_keys="Package.cfo_contact_id")
+    m_and_e_packages = sqla.orm.relationship(
+        "Package",
+        back_populates="m_and_e_contact",
+        foreign_keys="Package.m_and_e_contact_id",
+    )
+
+
+class Package(db.Model):
+    """Stores Package entities."""
+
+    __tablename__ = "package_dim"
+
+    id = sqla.Column(
+        GUID(),  # this should be UUIDType once using Postgres
+        default=uuid.uuid4,
+        primary_key=True,
+    )
+    package_name = sqla.Column(sqla.String, nullable=False, unique=True)
+    # TODO: should we limit string length here?
+    fund_type_id = sqla.Column(sqla.String, nullable=False)
+    # TODO: Check that we have organisation id defined at Package level AS WELL AS via contact.
```

Comment:
I'm not sure I understand this TODO? Do we need it going forwards or?

Reply:
yeah, but I'll re-write it for clarity

Reply:
Got it now